crates/vm/src/metrics/mod.rs

```diff
@@ -28,6 +28,8 @@ pub struct VmMetrics {
     #[allow(dead_code)]
     pub(crate) current_fn: FnBound,
     pub(crate) current_trace_cells: Vec<usize>,
+    // Maps opcode to number of (instances, reads, writes)
+    pub opcode_mem_accesses: BTreeMap<String, (usize, usize, usize)>,
```

Comment:
I think what we likely care more about is where these reads and write are located (which addresses), and what their alignment and block sizes look like.